apps/hash-graph/tests/integration/postgres/lib.rs

```diff
@@ -137,7 +137,9 @@ impl DatabaseTestWrapper {
 
             (data_type, metadata)
         });
-        store.create_data_types(data_types_iter).await?;
+        store
+            .create_data_types(data_types_iter, ConflictBehavior::Skip)
```

Comment:
Was `skip` chosen for the integration tests for a reason? I'm just curious :)

Reply:
Yes, it was quite annoying that I had to wipe the database every time I tried running the test. The possibility to skip those was quite nice.

Reply:
That's actually a pretty good reason, and something I've experienced as well. Thanks for explaining!